file_paths.md

```diff
@@ -0,0 +1,123 @@
+Puppet Installation Layout
+==========================
+
+This table specifies the file paths in a Puppet installation and the corresponding settings, denoted as `:setting`. The 3.x column shows the difference of the specification vs the actual 3.x implementation.
```

Comment:
I was thinking of this document as being the specification (which AIO is working on implementing, though not yet completely). And we're comparing the spec against what was implemented previously. But perhaps the distinction is more confusing than necessary.